src/main/java/appeng/container/implementations/ContainerCellWorkbench.java

```diff
@@ -240,8 +240,8 @@ public void partition()
 		{
 			if( i.hasNext() )
 			{
-				final ItemStack g = i.next().getItemStack();
-				g.setCount( 1 );
+				// TODO: check if ok
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)